README.md

```diff
@@ -1 +1,9 @@
-# kazoo-back
\ No newline at end of file
+# kazoo-back
+
+Levantar la BD
+
+1) sudo apt install postgresql.
```

Comment:
No hay chance de que me instale PostgreSQL en mi compu jajaja.
Docker for the win!

Reply:
jajajaja bueno pero yo como ya tenía postgres lo usé directo :see_no_evil: